solidity/contracts/converter/BancorConverter.sol

```diff
@@ -639,16 +631,15 @@ contract BancorConverter is IBancorConverter, TokenHandler, SmartTokenController
       * @param _sourceToken source reserve token contract address
       * @param _targetToken target reserve token contract address
       * @param _amount      amount of tokens to convert (in units of the source reserve token)
-      * @param _minReturn   if the conversion results in an amount smaller than the minimum return - it is cancelled, must be nonzero
       * @param _beneficiary wallet to receive the conversion result
       * 
       * @return amount of tokens received (in units of the target reserve token)
     */
-    function crossConvert(IERC20Token _sourceToken, IERC20Token _targetToken, uint256 _amount, uint256 _minReturn, address _beneficiary) internal returns (uint256) {
+    function crossConvert(IERC20Token _sourceToken, IERC20Token _targetToken, uint256 _amount, address _beneficiary) internal returns (uint256) {
         (uint256 amount, uint256 feeAmount) = getCrossReserveReturn(_sourceToken, _targetToken, _amount);
 
-        // ensure the trade gives something in return and meets the minimum requested amount
-        require(amount != 0 && amount >= _minReturn);
+        // ensure the trade gives something in return
+        require(amount != 0);

```

Comment:
@yudilevi : Doesn't this mean that we don't actually need to verify minReturn > 0 everywhere?